scripts/cargo-install-all.sh

```diff
@@ -90,59 +86,24 @@ cd "$(dirname "$0")"/..
 
 SECONDS=0
 
-if [[ $CI_OS_NAME = windows ]]; then
-  # Limit windows to end-user command-line tools.  Full validator support is not
-  # yet available on windows
-  BINS=(
-    cargo-build-sbf
-    cargo-test-sbf
-    solana
-    agave-install
-    agave-install-init
-    solana-keygen
-    solana-test-validator
-    solana-tokens
-  )
-  DCOU_BINS=()
-else
-  ./fetch-perf-libs.sh
-
-  DCOU_BINS=()
-  BINS=(
-    solana
-    solana-faucet
-    solana-genesis
-    agave-install
-    solana-keygen
-  )
-
-  if [[ -z "$publicRelease" ]]; then
-    BINS+=(
-      agave-validator
-      agave-watchtower
-      solana-gossip
-    )
-
-    DCOU_BINS+=(
-      agave-ledger-tool
-    )
-  fi
+source "$SOLANA_ROOT"/scripts/agave-build-lists.sh
 
+BINS=()
+DCOU_BINS=()
+if [[ -n "$validatorOnly" ]]; then
+  echo "Building binaries for net.sh deploys: ${AGAVE_BINS_NET_SH[*]}"
+  BINS+=("${AGAVE_BINS_NET_SH[@]}")
```

Comment:
It looks like this is doing the opposite of the old behavior. `-z "$validatorOnly"` in the old code meant that we only add `AGAVE_BINS_NET_SH` bins if we're NOT a validator-only build.

So instead this branch should be adding all of the validator-only bins and excluding `AGAVE_BINS_NET_SH`, which looks like:

* `solana`
* `solana-faucet`
* `solana-genesis`
* `agave-install`
* `solana-keygen`
* `agave-validator`
* `agave-watchtower`
* `solana-gossip`
* `agave-ledger-tool`

Or `AGAVE_BINS_END_USER` + `AGAVE_BINS_VAL_OP` + `AGAVE_BINS_DCOU`